walletnode/api/design/artworks.go

```diff
@@ -307,3 +340,190 @@ var RegisterTaskPayload = Type("RegisterTaskPayload", func() {
 	})
 	Required("taskId")
 })
+
+// SearchArtworkParams are query params to searchArtwork request
+var SearchArtworkParams = func() {
```

Comment:
Required fields must be listed
```go
Required("artist_name", ..........)
```

Reply:
This is a design decision depending on what user experience we want to provide on Wallet UI. I've asked several times. Can you help with this? I personally don't see any field that "should" be required. 
for minimum block, I put a default value.
@levkoburburas

Reply:
Without a required field, the search becomes useless. You must indicate what you are looking for.
I don't know, it isn't my scope. Try to be more persistent with question to @ayk @jeff. There is no other way.